ComfySigns/Patches/ZoneSystemPatch.cs

```diff
@@ -0,0 +1,20 @@
+﻿using HarmonyLib;
+
+using ComfyLib;
+
+using static ComfySigns.ComfySigns;
+using static ComfySigns.PluginConfig;
+
+namespace ComfySigns.Patches {
+  [HarmonyPatch(typeof(ZoneSystem))]
+  public class ZoneSystemPatch {
+    [HarmonyPostfix]
+    [HarmonyPatch(nameof(ZoneSystem.Awake))]
+    public static void ZoneSystemAwakePostfix(ZoneSystem __instance) {
```

Comment:
Remove the public modifier.

And since this is already within ZoneSystemPatch you can rename the method to `AwakePostfix` for brevity.
If you don't need access to the __instance you can also remove the argument.